group_project_v2/project_api/api_implementation.py

```diff
@@ -19,39 +16,18 @@
 SUBMISSION_API = '/'.join([API_PREFIX, 'submissions'])
 GROUP_API = '/'.join([API_PREFIX, 'groups'])
 COURSES_API = '/'.join([API_PREFIX, 'courses'])
+PROJECTS_API = '/'.join([API_PREFIX, 'projects'])
 
 
-DEFAULT_EXPIRATION_TIME = timedelta(seconds=10)
-
+def do_not_cache(reason):  # pylint:disable=unused-argument
```

Comment:
I'm not sure I understand how this works. The docstring says it's a no-op, and it just seems to contain a comment above the target function, which I would think could just be handled by adding hash-prefixed comments. What advantage does decorating the function actually give?